apps/dashboard/src/app/(dashboard)/api/auth/login/route.ts

```diff
@@ -0,0 +1,45 @@
+import { AUTH_COOKIE_NAME } from "@/constants/cookie";
+import { NextResponse } from "next/dist/server/web/spec-extension/response";
+import { cookies } from "next/headers";
+
+const threeDays = 3 * 24 * 60 * 60;
+
+export async function POST(request: Request) {
+  const cookieStore = cookies();
+
+  const body = await request.json();
+
+  // forward the request to the API server
+  const res = await fetch(
+    `${process.env.NEXT_PUBLIC_THIRDWEB_API_HOST}/v1/auth/login`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(body),
+    },
+  );
+
+  if (!res.ok) {
+    return new Response(null, {
+      status: 401,
```

Comment:
Should this return res.status instead of always 401?

Reply:
probably